### DISCUSSION TITLE
Any way to set the build target for tsconfig.json to ES6 or higher?

### DISCUSSION THREAD

**Discussion by tmsholy** [facebook/docusaurus#8958] [Q&A] [answered]
I would like to use some newer JavaScript features, and haven't found anything in the documentation about the build target. I have my own `tsconfig.json` for TypeScript, but setting it there is pointless since the documentation explicitly states:

> Docusaurus doesn't use this tsconfig.json to compile your project

I did not find the target defined anywhere, so I am guessing it uses the default `ES3`, but correct me if I'm wrong. Is there a way to change the target? I don't really see any reason to target anything older than ES6.

**Comment by Josh-Cena** [comment, marked as answer]
Hi, Docusaurus does not compile your code with tsc. It uses Babel and Webpack. Therefore you can change compilation using the browserslist config and/or the Babel config. See: https://docusaurus.io/docs/browser-support, https://docusaurus.io/docs/configuration#customizing-babel-configuration

**Reply by tmsholy** [reply]
Thank you!
